### DISCUSSION TITLE
Add port changing option in the clone operation

### DISCUSSION THREAD

**Discussion by vishalkadam47** [coollabsio/coolify#2290] [Feature Requests]
In Resource Operations while cloning a service it would be helpful to have a port changing option, or an option to assign a random unused port to avoid conflicts.
when I deploy an app using docker-compose each time I clone a service manually I need to check unused port and change the port to get successful deployment

**Comment by benjamin-steurer** [comment]
How do you currently do it? Just clone the repo, change port and then deploy?

**Comment by vishalkadam47** [comment]
after a clone, I manually change the port by editing the compose file from coolify and then deploy. while cloning an app if its possible to randomly change the unused port would also work.
